docs/user/07-80-available-presets.md

```diff
@@ -0,0 +1,37 @@
+# Available presets
+
+Function's resources and replicas as well as resources for image-building Jobs are based on presets. A preset is a predefined group of values. There are two groups of presets defined for a Function CR and include the presents for:
+
+- Function's resources
+- Image-building Job's resources
+
+## Configuration
```

Comment:
@kwiatekus 
Should I remove the whole Configuration section or only the last sentence that refers to the Chart doc that we removed?